tuxemon/monster.py

```diff
@@ -512,6 +512,12 @@ def set_level(self, level: int = 5) -> None:
         self.total_experience = self.experience_required()
         self.set_stats()
 
+        # Update moves
+        self.moves = []
```

Comment:
```suggestion
```

Reply:
This is called in a few other places. As there it is possible to learn new techniques via an action, I don't want to risk overwriting them.